### ISSUE TITLE
Only works for an admin account

### ISSUE THREAD

**Issue by brimur** [realriot/tvheadend-ng.bundle#48]
Thanks very much for this plugin, this is more of a question than an issue. I noticed when I was setting up an account in TVH for the NG plugin that only an admin account would work. Otherwise it would fail to list the channels. Is this by design or am I doing something wrong?

**Comment by realriot** [comment]
The plugin and the user needs the rights to pull some infos... If you have created a special user it should work.

**Comment by brimur** [comment]
Yes it works for any user but the Admin box has to be checked for that user or it fails to retrieve even the channel list.
